src/biogeochem/CNPhenologyMod.F90

```diff
@@ -907,13 +914,23 @@ subroutine CNSeasonDecidPhenology (num_soilp, filter_soilp       , &
                if (onset_gddflag(p) == 1.0_r8 .and. soilt > SHR_CONST_TKFRZ) then
                   onset_gdd(p) = onset_gdd(p) + (soilt-SHR_CONST_TKFRZ)*fracday
                end if
+               !separate into Arctic boreal and lower latitudes
+               if (onset_gdd(p) > crit_onset_gdd .and. abs(grc%latdeg(g))<45.0_r8) then
+                  onset_thresh=1.0_r8
+               else if (onset_gddflag(p) == 1.0_r8 .and. t10(p) > SHR_CONST_TKFRZ &
+                       .and. tmin10(p) > SHR_CONST_TKFRZ .and. ws_flag==1.0_r8 &
+                       .and. dayl(g)>(crit_dayl/2.0_r8) .and. snow5d(c)<0.1_r8) then
```

Comment:
Let's make sure we discuss this Wednesday.

Reply:
Let's make a new arctic seasonal deciduous parameterization with indices for the appropriate PFT, replace 
if <45 degrees... 
with 
if pft_# = ...
This could be done using parameter file?
@ekluzek can you provide @lmbirch89 with an example of how to subset the right pfts?

Reply:
While I don't understand exactly what this will look like, I'd like to make sure that we don't make #21 worse. So, to the extent possible, logic should depend on particular *properties* of PFTs (which can be set on the parameter file), rather than depending on particular PFT types. (But if this is specific to just a single PFT, then it could be okay to have logic that depends directly on that PFT type.)

Reply:
We talked about this in our CLM software meeting this morning. We agreed the best way to do this is to add a logical variable to the params file that says whether a given PFT is a cold plant type. So this would add a new variable called "is_cold_pft" to the params file that would be read in as a logical variable, and it would look like for example the "evergreen" variable which looks like this:

>         double evergreen(pft) ;
>                 evergreen:long_name = "Binary flag for evergreen leaf habit" ;
>                 evergreen:units = "logical flag" ;
>                 evergreen:coordinates = "pftname" ;
>                 evergreen:flag_meanings = "NON-evergreen evergreen" ;
>                 evergreen:flag_values = 0., 1. ;

When you use it inside of CNPhenology you would access the "ivt(p)" element of it for a given patch p.

So something like...

`if ( is_cold_pt(ivt(p)) ) then`